src/v0/destinations/pardot/networkHandler.js

```diff
@@ -73,13 +49,15 @@ const pardotRespHandler = (destResponse, stageMsg, stage) => {
     return response;
   }
   const statusAndStats = getStatusAndStats(errorCode, stage);
-  throw new ErrorBuilder()
-    .setStatus(statusAndStats.status)
-    .setDestinationResponse(response)
-    .setMessage(`Pardot: ${response.err} ${stageMsg}`)
-    .setAuthErrorCategory(getAuthErrCategory(errorCode))
-    .setStatTags(statusAndStats.stats)
-    .build();
+  const destinationStatusCode = statusAndStats?.status || 400;
+  throw new NetworkError(
+    `Pardot: ${response.err} ${stageMsg}`,
+    destinationStatusCode,
+    {
+      [tags.TAG_NAMES.ERROR_TYPE]: getDynamicErrorType(destinationStatusCode)
+    },
+    response
+  );
```

Comment:
We are not setting AuthErrCategory for this?
This is Oauth right?

Reply:
```
throw new ErrorBuilder()
    .setStatus(statusAndStats.status)
    .setDestinationResponse(response)
    .setMessage(`Pardot: ${response.err} ${stageMsg}`)
    .setAuthErrorCategory(getAuthErrCategory(errorCode))
    .setStatTags(statusAndStats.stats)
    .build();
```
We are sending `authErrorCategory` via a method right ?!

Reply:
I didn't understand for NetworkError(), the last argument is `authErrorCategory`

Reply:
can you check it now?